lib/base.js

```diff
@@ -47,7 +47,7 @@ class BaseModel {
             }
         };
 
-        return nodeify(this.datastore.get, config, callback);
+        return nodeify(this.datastore, this.datastore.get, config, callback);
```

Comment:
I guess we'd still need to determine if the second arg is a function or a string.

Alternatively, there could be a `nodeify.apply(context, fn, ...)` that would be more explicit.